tests/units/aps/test_utils.py

```diff
@@ -57,10 +58,10 @@ def test_save_file_in_s3():
     assert key == expected_key
 
 
-def test_split_json():
-    doi = "10.1103/PhysRevLett.126.153601"
-    today = datetime.now().strftime("%Y-%m-%dT%H:%M")
+@freeze_time("2023-12-04")
+def test_split_json(monkeypatch):
+    monkeypatch.setenv("TZ", "GMT")
```

Comment:
Unfortunately, this will not work either since if we change, the server's timezone might be different. We don't need this https://github.com/cern-sis/workflows/commit/d2f9e84841cd9ae01ce0485a968a13c47d44a7b5#diff-70b5a470d6615a3d2f378c02dd505cac0aca0f865f188a88f95d0a69912bd630R189 in the first place. If we remove it, it will work with just mocking with freeze time.

Reply:
I wanted to be consistent with run_id namings, like run_id=manual__2023-12-04T10:01:21+00:00